lib/galaxy/datatypes/speech.py

```diff
@@ -0,0 +1,84 @@
+import re
+
+from galaxy.datatypes.text import Text
+from galaxy.datatypes.metadata import MetadataElement, ListParameter
+
+
+class TextGrid( Text ):
+    """Praat Textgrid file for speech annotations
+
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname('1_1119_2_22_001.TextGrid')
+    >>> TextGrid().sniff(fname)
+    True
+
+    >>> fname = get_test_fname('drugbank_drugs.cml')
+    >>> TextGrid().sniff(fname)
+    False
+
+    """
+
+    file_ext = "TextGrid"
```

Comment:
```suggestion
    file_ext = "textgrid"
```

Reply:
Huh, made change and forgot to stage. This works, thanks.